backends/platform/sdl/psp2/psp2.mk

```diff
@@ -3,13 +3,17 @@ PSP2_EXE_STRIPPED := scummvm_stripped$(EXEEXT)
 $(PSP2_EXE_STRIPPED): $(EXECUTABLE)
 	$(STRIP) --strip-debug $< -o $@
 
-psp2vpk: $(PSP2_EXE_STRIPPED)
+psp2vpk: $(PSP2_EXE_STRIPPED) $(PLUGINS)
 	rm -rf psp2pkg
 	rm -f $(EXECUTABLE).vpk
 	mkdir -p psp2pkg/sce_sys/livearea/contents
 	mkdir -p psp2pkg/data/
 	mkdir -p psp2pkg/doc/
 	vita-elf-create $(PSP2_EXE_STRIPPED) $(EXECUTABLE).velf
+ifdef DYNAMIC_MODULES
+	printf '\0' | dd of=$(EXECUTABLE).velf bs=1 seek=16 count=1 conv=notrunc
+	dd if=$(EXECUTABLE).velf bs=1 skip=24 count=4 | dd of=$(EXECUTABLE).velf bs=1 seek=64 count=4 conv=notrunc
```

Comment:
This feels slightly fragile. Would it be possible to extend vita-elf-create to do this instead of modifying it by hand?

Reply:
Yes, it is planned. That's a reason why this PR is in draft.
I want to make sure it works on real Vita before considering a PR on vita toolchain.